bwell-typescript-react/.env.example

```
@@ -1,2 +1,6 @@
 VITE_DEFAULT_KEY=""
-VITE_DEFAULT_OAUTH_CREDS=""
\ No newline at end of file
+VITE_DEFAULT_OAUTH_CREDS=""
+VITE_BAD_KEY="blahblah"
+VITE_BAD_OAUTH_CREDS="badoauthcreds"
+VITE_WELL_FORMED_KEY_BAD_ENV="eyJyIjoibm9uc2Vuc2UiLCJlbnYiOiJub2VudiIsImtpZCI6ImJsYWhibGFoIn0="
```

```suggestion
VITE_WELL_FORMED_KEY_BAD_ENV=""
```
Might suggest removing. Not critical but does reveal some about our key set up. May be good to purge it from the history of this branch as well.

Don't believe there's any other near creds data in the PR but may be good to double check.

---

Yeah, this was the only one. I pushed a fix to empty the string, think squashing when merging will remove it from history?

---

For the most part. An interested party can likely dig it out but isn't critical. Is technically something we use in the FE anyways so it's exposed elsewhere. Require creds along with the key for it to be useful.